Pipfile.lock

```diff
@@ -0,0 +1,206 @@
+{
```

Comment:
It's not common practice for libraries to ship a lockfile. Lockfiles are great for end-users who want to have a reproducible build, with specific versions of dependencies. In this case, we should not include the lockfile.